designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiHttpService.scala

```diff
@@ -247,20 +268,17 @@ class ScenarioActivityApiHttpService(
     }
 
   private def toDto(scenarioComment: ScenarioComment): Dtos.ScenarioActivityComment = {
-    scenarioComment match {
-      case ScenarioComment.Available(comment, lastModifiedByUserName, lastModifiedAt) =>
-        Dtos.ScenarioActivityComment(
-          content = Dtos.ScenarioActivityCommentContent.Available(comment),
-          lastModifiedBy = lastModifiedByUserName.value,
-          lastModifiedAt = lastModifiedAt,
-        )
-      case ScenarioComment.Deleted(deletedByUserName, deletedAt) =>
-        Dtos.ScenarioActivityComment(
-          content = Dtos.ScenarioActivityCommentContent.Deleted,
-          lastModifiedBy = deletedByUserName.value,
-          lastModifiedAt = deletedAt,
-        )
+    val content = scenarioComment match {
```

Comment:
I'm not sure if this is the correct place to do so. 

I assume that we allowed (for some reason) activity to be saved with AvailableComment with an empty string. 
It means that we should:
1. forbid saving it (and probably introduce NonEmptyString instead of String for the field that holds the comment content)
2. write a migration

Or we could:
1. forbid saving it (or save it with no comment option)
2. encapsulate the fact that in the DB we could have available comments that are empty, and convert them to non-available comments in the repo.

Handling it at the service level it too late IMO.

Reply:
Added handling on repository level, both on save and on read.

Reply:
is the second case correct?
```
      case ScenarioComment.WithContent(comment, _, _) =>
        Dtos.ScenarioActivityCommentContent.Available(comment)
      case ScenarioComment.WithoutContent(_, _) | ScenarioComment.WithContent(_, _, _) =>
        Dtos.ScenarioActivityCommentContent.NotAvailable
```

Reply:
fixed